h/features/models.py

```diff
@@ -105,3 +112,57 @@ def remove_old_flags(cls):
 
     def __repr__(self):
         return '<Feature {f.name} everyone={f.everyone}>'.format(f=self)
+
+
+class Cohort(Base, mixins.Timestamps):
```

Comment:
I don't mind the model class name not matching the table name, as long as it's easy to figure out which ones belong together. The one downside I can think off is that it makes it harder to rename the package as running a migration to rename a table in production without downtime is not very easy.